docs/API.md

```diff
@@ -140,8 +140,8 @@ import { GeoJSONLayer } from "react-mapbox-gl";
 
 ### Properties
 - **data** *(required)* : `String | Object` The url to the geojson file or the geojson file itself.
-- **lineLayout** | **symbolLayout** | **circleLayout** | **fillLayout** : `Object` Layer layout informations. [mapbox layout api](https://www.mapbox.com/mapbox-gl-style-spec/#layer-layout)
-- **linePaint** | **symbolPaint** | **circlePaint** | **fillPaint** : `Object` Paint informations. [mapbox paint api](https://www.mapbox.com/mapbox-gl-style-spec/#layer-paint)
+- **lineLayout** | **symbolLayout** | **circleLayout** | **fillLayout** | **fillExtrusionLayout** : `Object` Layer layout informations. [mapbox layout api](https://www.mapbox.com/mapbox-gl-style-spec/#layer-layout)
```

Comment:
(Unrelated to this line)

can you remove the example `yarn.lock` file? I know it's unnecessary and we all love yarn, but unless the entire project supports it, we might not want to create any confusion ;)